legal-api/src/legal_api/models/address.py

```diff
@@ -32,7 +32,8 @@ class Address(db.Model):  # pylint: disable=too-many-instance-attributes
 
     MAILING = 'mailing'
     DELIVERY = 'delivery'
-    ADDRESS_TYPES = [MAILING, DELIVERY]
+    FURNISHING = 'furnishing'
+    ADDRESS_TYPES = [MAILING, DELIVERY, FURNISHING]
```

Comment:
I know i have this in the design but I'm actually still thinking about how we will model the address used at the time the dissolution letters need to go out.

Can we leave this out for now?